src/index.html

```diff
@@ -11,6 +11,26 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- <h1>Search bar airbnb</h1> -->
```

Comment:
Remove comment.

Reply:
Done. But this line
<h1>Search bar airbnb</h1>
crash all tests